### PR TITLE
Favorites page

### PR DESCRIPTION
**Pull request by eileenfuerstenau**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,44 +1,16 @@
...
       </AppLayout>
     </>
   )
 }
 
 const AppLayout = styled.div`
-  display: grid;
-  gap: 0.2%;
-  padding: 1%;
+  margin: 0;
```
Comment (dride-for-sure) [inline src/components/App.js:15]: Isnt this the default value for a div? So if there are no globalstyles for all divs, which should be avoided (?), you could remove the margin here.

File: src/components/BookCardsPage.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+              title={card.title}
+              author={card.author}
+              description={card.content}
+              onBookmarkClick={handleBookmarkClick}
+              bookmarkedBooks={bookmarkedBooks}
+            />
+          ))}
+        <NoFavoritesStatement>
+          {bookmarkedBooks.length === 0 && booksShown === 'favorites'
+            ? 'Du hast noch keine Favoriten.'
```
Comment (BerePendzialek) [inline src/components/BookCardsPage.js:58]: wow. it worked! Congrats ;)

File: src/components/App.js (hunk 1 above)
Comment (macfrei) [reply src/components/App.js:15]: I think so too.

File: src/components/BookCard.js [hunk 3]
```diff
@@ -64,7 +64,7 @@ BookCard.propTypes = {
   author: PropTypes.string,
   description: PropTypes.string,
   onBookmarkClick: PropTypes.func,
-  bookmarkedBooks: PropTypes.string,
+  bookmarkedBooks: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
```
Comment (macfrei) [inline src/components/BookCard.js:67]: Nice! :)

File: src/components/BookCard.js [hunk 4]
```diff
@@ -78,24 +78,23 @@ const Card = styled.section`
...
-  font-weight: normal;
-  font-size: 90%;
+  padding: 15px 10px;
 `
 const Title = styled.h2`
   font-weight: bold;
   font-size: 100%;
   padding-right: 40px;
 `
+const Author = styled.h2`
```
Comment (macfrei) [inline src/components/BookCard.js:90]: Are `Title` and `Author` hierarchically the same? As in are they both equally important? If not, then `Author` should be a `h3`-element. Just something to think about :)

File: src/components/BookCardsPage.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        book => book !== currentBook
+      )
+    } else {
+      bookmarkedBooksArray = [...bookmarkedBooks, currentBook]
+    }
+    setBookmarkedBooks(bookmarkedBooksArray)
+  }
+
+  return (
+    <>
```
Comment (macfrei) [inline src/components/BookCardsPage.js:23]: You don't need these fragments, your `CardspageLayout` is sufficient.

File: src/components/BookCardsPage.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+      <CardspageLayout>
```
Comment (macfrei) [inline src/components/BookCardsPage.js:24]: Maybe either `CardPageLayout` or:
```suggestion
      <PageLayout>
```

**Review by macfrei — COMMENTED** [review]
Good job! Just added a few comment you could consider :)

File: src/components/App.js (hunk 1 above)
Comment (eileenfuerstenau) [reply src/components/App.js:15]: Thanks, i removed the margin here

File: src/components/BookCardsPage.js (hunk 2 above)
Comment (eileenfuerstenau) [reply src/components/BookCardsPage.js:58]: Haha yeees, Paula helped me trough

File: src/components/BookCardsPage.js (hunk 6 above)
Comment (eileenfuerstenau) [reply src/components/BookCardsPage.js:24]: Thanks, i named it CardPageLayout now :-)

File: src/components/BookCard.js [hunk 7]
```diff
@@ -64,7 +78,7 @@ BookCard.propTypes = {
   author: PropTypes.string,
   description: PropTypes.string,
   onBookmarkClick: PropTypes.func,
-  bookmarkedBooks: PropTypes.string,
+  bookmarkedBooks: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
```
Comment (marcushohlbein) [inline src/components/BookCard.js:81]: that's a cool solution

**Review by marcushohlbein — APPROVED** [review]
Very good and clean code.

**Pull request merged**